### ISSUE TITLE
config file modifications don't work

### ISSUE THREAD

**Issue by mattiasecchiero** [open-mmlab/mmsegmentation#3007]
Hy, when i try to fine-tune DeepLabV3+ trained with ADE20k some of the new config parameters are not updated. 
For example I change the "max_iters" of the train_cfg dict and the  "end" and "eta_min" of the param_scheduler dict but the latter one is not changing.
I want to change eta_min to 1e-06 but it keeps staying at 1e-04 value during training. I don't understand why since the other changes are updated

I am running this command : $ bash tools/dist_train.sh finetuning_deeplabv3plus_r101-d8_4xb4-80k-512x512.py 4 --resume --cfg-options load_from=./work_dirs/deeplabv3plus_r101-d8_4xb4-160k_ade-512x512/iter_160000.pt

where the config file is the same one used for training deeplabv3+ on ade20k but with different eta_min and max number of iterations. The check point loaded is the regular training of deeplab on ade20k for 160k iterations.

Substantially i want to fine-tune the model for other 80k iterations on a modified version of ade20k. The training starts correctly but the learning rate stays fixed to 1e-04 value and don't decrease.
